src/cache-updater/index.ts

```diff
@@ -0,0 +1,132 @@
+import fs from 'fs';
+import path from 'path';
+import { promisify } from 'util';
+import { Upload } from '@aws-sdk/lib-storage';
+import { S3Client } from '@aws-sdk/client-s3';
+
+import { executeCliCommand } from '../commands/src/helpers';
+
+const readdirAsync = promisify(fs.readdir);
+const SNOOTY_CACHE_BUCKET_NAME = process.env.SNOOTY_CACHE_BUCKET_NAME;
+
+if (!SNOOTY_CACHE_BUCKET_NAME) throw new Error('ERROR! SNOOTY_CACHE_BUCKET_NAME is not defined');
+
+async function cloneDocsRepo(repoName: string, repoOwner: string) {
```

Comment:
Good point! From what I understand though, these cache files will only be for the master/main branch I believe... I'll double check though to confirm.

Reply:
discussed offline![ this doc](https://docs.google.com/document/d/1ZJY8deWAqmAVrsvsb1emImerhqOT8q8EM4UJaPhCyWU/edit#heading=h.wyrxqhxvy787) points out this will be for the master/main branch of the repo